pandas/tests/io/test_common.py

```diff
@@ -360,3 +361,65 @@ def test_unknown_engine(self):
             df.to_csv(path)
             with pytest.raises(ValueError, match="Unknown engine"):
                 pd.read_csv(path, engine="pyt")
+
+
+@pytest.mark.parametrize(
+    "reader",
+    [
+        pd.read_csv,
+        pd.read_table,
+        pd.read_fwf,
+        pytest.param(pd.read_excel, marks=[td.skip_if_no("xlrd")]),
+        pytest.param(
+            pd.read_json,
+            marks=[
+                pytest.mark.xfail(
+                    reason=(
+                        "Needs to distinguish between a path to read "
+                        "vs a string to parse"
```

Comment:
@WillAyd any thoughts on this?  I think I saw a semi-recent discussion about trying to guess whether a string is supposed to be a path or actual content.

Reply:
Yea this was a discussion with @datapythonista and @gfyoung . I don't have a strong point of view in any direction but would be against jumping through a lot of hoops to disambiguate.

I guess if we cared to disambiguate could probably just look for `[`, `{`, `"` or any leading number (ignoring whitespace) to start as those are the requirements for JSON

Reply:
I would leave the JSON disambiguation alone for now.  I'm still not really that inclined to change at the moment.

Reply:
Looking through old issues, there are a lot that revolve around disambiguating paths vs content, e.g. #5924 for pickle.  (one clever suggestion was to have `pd.read_pickles`, `pd.read_jsons`, etc as analogues to the stdlib `json.loads`)

This doesn't need to be resolved for this PR, but I think we're going to need to have a decision about whether/when/how to guess if a string is a path or not.